.github/CONTRIBUTING.md

```diff
@@ -104,6 +104,12 @@ Most of the tests in PyTorch Lightning train a trial MNIST model under various t
 
 ## Guidelines
 
+**Hint:**
```

Comment:
I would add this under ### Test cases above. 

Also, can you add a link to the boring model as a base test case?

Reply:
this is also for building docs which is not test